spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -116,13 +116,13 @@ Fault Tolerance 3.0 is now dependent on Jakarta EE 8 technologies.
  - https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/3.0
 
 [[mp-health-check]]
-=== MicroProfile Health 3.0
+=== MicroProfile Health 3.1
 
 MicroProfile Health provides the ability to expose the health of an application
 to the underlying platform (ex: Kubernetes)
 
  - http://microprofile.io/project/eclipse/microprofile-health
- - https://github.com/eclipse/microprofile-health/releases/tag/3.0
+ - https://github.com/eclipse/microprofile-health/releases/tag/3.1
```

Comment:
here it should be final? Not 3.1-RC3?

Reply:
It is possible for the sake of clarity, but not necessary. What is your preference?

Reply:
I would keep it consistent so 3.1-RC3. We should write up some scripts/automation do these updates with a single command in the future.